41. First Missing Positive/solution.cpp

```diff
@@ -0,0 +1,60 @@
+class Solution

```

Comment:
Add the Author's name, TC, SC, and Brief approach used...See other solution files in the repo.

Reply:
yeah sure..